docs/api.rst

```diff
@@ -422,45 +422,41 @@ Attributes
    ModelComponent.logger
    ModelComponent.model_root
 
-ModelRegionComponent
-====================
+SpatialModelComponent
+=====================
+
 
 .. autosummary::
    :toctree: _generated
 
-   ModelRegionComponent
+   SpatialModelComponent
 
 Attributes
 ----------
 
 .. autosummary::
    :toctree: _generated
 
-   ModelRegionComponent.data
-   ModelRegionComponent.model
-   ModelRegionComponent.crs
-   ModelRegionComponent.bounds
-   ModelRegionComponent.data_catalog
-   ModelRegionComponent.logger
-   ModelRegionComponent.model_root
-
-General Methods
----------------
+   SpatialModelComponent.model
+   SpatialModelComponent.data_catalog
+   SpatialModelComponent.logger
```

Comment:
You are right, but I also think we are not consistent at the moment. We decided a couple of weeks ago that we should go for `_data_catalog`, because a normal user should retrieve it via `Model.data_catalog` (which is the same reference). Plugin developers can still retrieve this protected property.

But now for `create_region`, we have decided that it should be public, because of plugin developers. We should discuss this again and be consistent.

Reply:
The function moved to a workflow, so it's now public anyways